src/transaction/log_page_buffer.c

```diff
@@ -8766,6 +8771,39 @@ logpb_restore (THREAD_ENTRY * thread_p, const char *db_fullname, const char *log
       try_level = (FILEIO_BACKUP_LEVEL) (try_level - 1);
     }
 
+  // Incremental backup volumes often do not include volume header pages.
+  // Accordingly, it is necessary to set unlinked volumes after all volume header pages are restored.
+  // *INDENT-OFF*
+  for (const auto &unlinked_itr : unlinked_volinfo)
```

Comment:
Nit. it's not an iterator.
```suggestion
  for (const auto &volinfo_pair : unlinked_volinfo)
```

Reply:
Thank you. But, I won't edit it to merge right away.